HEN_HOUSE/src/get_inputs.mortran

```diff
@@ -1690,6 +1713,28 @@ write(i_errors,*)
 ' ******************** end input transport parameter *********************** ';
 write(i_errors,*);
 
+"Check if EADL relaxation requested. Note that original relaxation"
+"algorithm using <M> and <N> is only turned ON for all regions."
+"Moved passed the $TURN-ON/OFF-IN-REGIONS statement to catch the"
```

Comment:
Thanks a bunch @rtownson for reviewing! Updating the documentation is the next step.